tests/codegen/float/f128-algebraic.rs

```diff
@@ -0,0 +1,40 @@
+// Verify that algebraic intrinsics generate the correct LLVM calls for f128
+
+#![crate_type = "lib"]
+#![feature(f128)]
+#![feature(float_algebraic)]
+
+// CHECK-LABEL: fp128 @f128_algebraic_add(
+#[no_mangle]
+pub fn f128_algebraic_add(a: f128, b: f128) -> f128 {
+    // CHECK: fadd reassoc nsz arcp contract fp128 {{(%a, %b)|(%b, %a)}}
```

Comment:
The addition and multiplication cases both end up as `%b, %a` rather than `%a, %b` which surprised me but isn't incorrect. I opted to allow either in case behavior changes in the future.